docs/configuration.md

```diff
@@ -1629,8 +1629,10 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.blacklist.killBlacklistedExecutors</code></td>
   <td>false</td>
   <td>
-    (Experimental) If set to "true", allow Spark to automatically kill, and attempt to re-create,
-    executors when they are blacklisted.  Note that, when an entire node is added to the blacklist,
+    (Experimental) If set to "true", allow Spark to automatically kill the
+    executors when they are blacklisted on fetch failure or blacklisted for the entire application, 
+    as controlled by spark.blacklist.application.*.  
+    Note that, when an entire node is added to the blacklist,
```

Comment:
very minor nit, can you put this on previous line and wrap text accordingly

Reply:
ok, i checked the README.md and it did not spoil the indentation. No issues :)